modules/navier_stokes/doc/content/source/physics/WCNSFVFlowPhysics.md

```diff
@@ -0,0 +1,60 @@
+# WCNSFVFlowPhysics
```

Comment:
I am just looking at it from a user's point of view. I am on the source site and see the available options. I click on WCNSFVFlow and think oh okay so this is what I have to add. But no, it is actually just Flow. I'd advocate for consistency one way or another. What was the conclusion here, @lindsayad ?
Also, I think there might be an alias option in registration, not sure that is possible/useful in this case.

Reply:
I don't see someone writing a physics for CNSFV or finite elements any time soon from a practical standpoint. I agree that this should be consistent. And I think it should be `Flow`. This is where we're spending the overwhelming majority of our resources and where we'll be spending it in the foreseeable future. We should have a canonical flow object. I think it's ok if in in the future someone comes along and writes `CompressibleFlow` that lives alongside `Flow`. Or `HDGFlow` that lives alongside `Flow`

Reply:
The page has been renamed to "Navier Stokes Flow / WCNSFVFlowPhysics"
The syntax in the input has been changed from [WCNSFVFlow] to [Flow]

Given the large differences between fully coupled and segregated setups, we will likely have to write another physics (for readability) in a shorter time frame than expected. This probably means a NSFVFlowPhysicsBase, with WCNSFV and "some segregated prefix" FlowPhysics derived classes. Renaming WCNSFVFlowPhysics in the code now is not time well spent

Reply:
Okay, if the documentation and the syntax are consistent it is good enough for me. Thanks for the changes! Yes, we should add the segregated solver at some point. I thought we could have added it to the same physics with incorporating the executioner but if you want to avoid large physics we can probably write a new one.

Reply:
If you keep the kernels with rougjly the same parameters in the segregated solver we can shimmy something that will be reasonable to maintain.

Reply:
It would be my preference. We ll have to look at how different the input file looks like really. The action (physics) really mirror the input file

Reply:
I guess I'm pretty inconsistent on this. We'll see!

Reply:
A few `git mv` and then one `git sed WCNSFVFlowPhysics Flow`. Should take less than 5 minutes. And then the next physics could be `FlowSegregated` or whatever. Or if we think the `FlowSegregated` will the most used, then the current one could be `FlowMonolithic` and the next could be `Flow`. But I won't make this a hill that I'll die on. One thing that does need to be fixed though is that `/` characters do not get rendered on the source index page. I would suggest removing "/ WCNSFVFlowPhysics" from "Navier Stokes Flow / WCNSFVFlowPhysics"

Reply:
if it shows as "Navier Stokes Flow WCNSFVFlowPhysics" it's still fine for me

do you have an issue for / not rendering? I might create one I think we should address it this summer

Reply:
I did not see one. I created https://github.com/idaholab/moose/issues/27525. Ok stubborn goose, we'll keep things as they are